packages/neutrino-middleware-start-server/index.js

```diff
@@ -1,3 +1,13 @@
 const StartServerPlugin = require('start-server-webpack-plugin');
 
-module.exports = ({ config }, options) => config.plugin('start-server').use(StartServerPlugin, [options.name]);
+module.exports = (neutrino, options) => {
```

Comment:
This could probably be shortened to:

```js
module.exports = (neutrino, options) => neutrino.config
  .plugin('start-server')
    .use(StartServerPlugin, [{
      name: options.name,
      nodeArgs: neutrino.options.debug ? ['--inspect'] : []
    }]);
```

Thoughts?